templates/tab/js/default/package.json

```diff
@@ -5,13 +5,17 @@
     "dependencies": {
         "@fluentui/react-northstar": "^0.54.0",
         "@microsoft/teams-js": "^1.9.0",
+        "@microsoft/microsoft-graph-client": "^3.0.1",
         "@microsoft/teamsfx": "^0.5.1",
         "axios": "^0.21.1",
         "msteams-react-base-component": "^3.1.0",
         "react": "^16.14.0",
         "react-dom": "^16.14.0",
         "react-router-dom": "^5.1.2",
-        "react-scripts": "^4.0.3"
+        "react-scripts": "^4.0.3",
+        "@microsoft/mgt-react": "2.3.2-next.teamsfx.f5fc7d2",
+        "@microsoft/mgt-element": "2.3.2-next.teamsfx.f5fc7d2",
+        "@microsoft/mgt-teamsfx-provider": "2.3.2-next.teamsfx.f5fc7d2"
```

Comment:
Better to sort dependencies by alphabet.

Reply:
sure~

Reply:
Updated